### ISSUE TITLE
Cannot login with normal user

### ISSUE THREAD

**Issue by thesonpb** [AOT-Technologies/forms-flow-ai#1508]
### Please provide information about your setup

- formsflow.ai Version: [window desktop](https://aot-technologies.github.io/forms-flow-installation-doc/Pages/PC/windows/onWindows.html)
- Browser: Google Chrome
- Browser version: 114.0.5735.199

### Steps to Reproduce

1. Login with admin account (admin/changeme)
2. Create new user, create credential
3. Login with the new user credential on [my-ip]:8080

### Observed Behaviour

It redirect to localhost:3000 and then go back to [my-ip]:8080 with the login screen.

### Expected Behaviour

It redirect to localhost:3000/form to create form

**Comment by sumesh-aot** [comment]
@thesonpb Does the new user have one of /formsflow-designer, /formsflow-client or /formsflow-reviewer group assignment ? Application would check for these authorization.

**Comment by thesonpb** [comment]
@sumesh-aot yes, the new user is assigned to /formsflow-designer group, it just redirect me to localhost:3000 and then go back to the blank login page

**Comment by alanraju-aot** [comment]
@thesonpb  Can you send the screenshots on how you have created, because we are able to create new user and login with those new credentials.

![image](https://github.com/AOT-Technologies/forms-flow-ai/assets/94040192/92e75d3d-dded-40d1-be46-f383c2c4f460)

**Comment by alanraju-aot** [comment]
![image](https://github.com/AOT-Technologies/forms-flow-ai/assets/94040192/5747d390-0e85-425c-a0c9-16ab3c7dc7ce)
